browser/src/Services/QuickOpen.ts

```diff
@@ -55,46 +62,60 @@ export class QuickOpen {
                 const files = execSync(overriddenCommand.replace("${search}", ""), { cwd: process.cwd() })
                     .toString("utf8")
                     .split("\n")
-                this._showMenuFromFiles(files)
+                this._showMenuFromFiles(files, "file-text-o")
                 return
             } catch (e) {
                 Log.warn(`'${overriddenCommand}' returned an error: ${e.message}\nUsing default file list`)
             }
         }
 
         // Default strategy
-        //  If git repo, use git ls-files
+        //  If in exec directory, show bookmarks to change cwd to
+        //  Otherwise, If git repo, use git ls-files
         //  Otherwise, find all files recursively
         const openPromise = Git.isGitRepository()
             .then((isGit) => {
+                if (process.execPath === process.cwd()) {
```

Comment:
We might want to make this a helper method, like `isInstallDirectory(process.cwd())` - we might need to differentiate the logic a bit between platform. I believe for Windows this should work, but for OSX and Linux this logic might not be quite correct. Refactoring it to a method will help encapsulate this without the if getting too crazy.